learn2learn/vision/datasets/fgvc_aircraft.py

```diff
@@ -115,7 +115,8 @@ def __init__(
         self.transform = transform
         self.target_transform = target_transform
         self.bounding_box_crop = bounding_box_crop
-        self._bookkeeping_path = os.path.join(self.root, 'fgvc-aircraft-' + mode + '-bookkeeping.pkl')
+        self._bookkeeping_path = os.path.join(
+            self.root, 'fgvc-aircraft-' + mode + '-bookkeeping.pkl')
```

Comment:
I recommend using pathlib.Path for general file system operations. It's a lot more versatile and intuitive.